tests/Unleash.Tests/Streaming/StreamingFeatureFetcherTests.cs

```diff
@@ -1,62 +1,13 @@
 using Unleash;
-using Unleash.Streaming;
-using Unleash.Communication;
 using NUnit.Framework;
-using Unleash.Metrics;
-using Yggdrasil;
-using Unleash.Scheduling;
 using LaunchDarkly.EventSource;
-using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Http;
 using NUnit.Framework.Internal;
-using Microsoft.AspNetCore.TestHost;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Routing;
-using Microsoft.Extensions.DependencyInjection;
 using System.Diagnostics;
 using Unleash.Tests.Mock;
+using static Unleash.Tests.StreamingServer;
```

Comment:
`using static` here imports all static methods on the static class StreamingServer so we can invoke them as static functions here
ie.
`GetStreamingTestServerClient` and `WriteEvents` instead of `StreamingServer.GetStreamingTestServerClient` and `StreamingServer.WriteEvents`